.npmrc

```diff
@@ -1,3 +0,0 @@
-scripts-prepend-node-path=true
-@openforis:registry=https://npm.pkg.github.com
-always-auth=true
```

Comment:
don't delete this file, it doesn't contain any personal information and allows every user that dowloads the source code to build it